spec/latest/common/terms.html

```diff
@@ -212,6 +212,14 @@
       A <a>prefix</a> is a <a>term</a> that expands to a vocabulary <a>base IRI</a>. It
       is typically used along with a <em>suffix</em> to form a <a>compact IRI</a> to create an IRI
       within a vocabulary.</dd>
+    <dt><dfn>processing mode</dfn></dt><dd>
+      The processing mode defines how a JSON-LD document is processed.
+      By default, all documents are assumed to be conformat with
+      <a data-cite="JSON-LD-20140116">JSON-LD 1.0</a> [[!JSON-LD-20140116]]. By defining
+      a different version using the <code>@version</code> member in a
```

Comment:
I don't feel that strongly, so I'm ok with going forward with the numeric proposal for now given its necessity for causing a 1.0 processor to halt.